Classes/ViewHelpers/RenderValuesViewHelper.php

```diff
@@ -30,19 +30,34 @@ class RenderValuesViewHelper extends AbstractViewHelper
     protected $escapeOutput = false;
 
     /**
-     * @param RootRenderableInterface $renderable If specified, only the values of the given renderable are rendered, otherwise all form elements are rendered
-     * @param FormRuntime $formRuntime If not set, the Form Runtime will be fetched from the View, which only works within the FluidFormRenderer
-     * @param string $as
+     * Initialize the arguments.
+     *
+     * @return void
+     * @throws \Neos\FluidAdaptor\Core\ViewHelper\Exception
+     */
+    public function initializeArguments()
+    {
+        parent::initializeArguments();
+        $this->registerArgument('renderable', RootRenderableInterface::class, 'Relative Fusion path to be rendered');
+        $this->registerArgument('formRuntime', FormRuntime::class, 'Relative Fusion path to be rendered');
+        $this->registerArgument('as', 'string', 'Relative Fusion path to be rendered', false, 'formValue');
+    }
+
+    /**
      * @return string the rendered form values
      */
-    public function render(RootRenderableInterface $renderable = null, FormRuntime $formRuntime = null, $as = 'formValue')
+    public function render(): string
     {
-        if ($formRuntime === null) {
+        if ($this->hasArgument('formRuntime')) {
```

Comment:
Works, because  `hasArgument()` uses isset() which also checks for null. Chapeau!